src/documents/CoreAPI.md

```diff
@@ -0,0 +1,635 @@
+will be removed from the source structure.
+* `target {Array|Object}` [optional] A target object of the same type as `source`, which will
+receive any objects removed from it.
+* Returns: `{Array|Object}` Either `target`, containing the removed elements, if it was supplied, or else `source`
+modified by the operation of removing the matched elements.
+
+
+### fluid.getMembers(holder, path)
+
+Extracts a particular member from each top-level member of a container, returning a new container of the same type.
+* `holder {Array|Object}` The container to be filtered
+* `path {String|Array of String}` An [EL path](FrameworkConcepts.md#el-paths) to be fetched from each top-level member
+* Returns: `{Array|Object}` A container of the same type as `holder`, with the members of `holder` deferenced by `path`.
+
+
+### fluid.filterKeys(toFilter, keys[, exclude])
+
+Accepts an object to be filtered, and a list of keys. Either all keys not present in
+the list are removed, or only keys present in the list are returned. The suppled object *** is not modified *** but will be shallow cloned by this operation.
+ 
+* `toFilter {Array|Object}` The object to be filtered - this will be NOT modified by the operation
+* `keys {Array of String}` The list of keys to operate with
+* `exclude {Boolean}` [optional] If `true`, the keys listed are removed rather than included
+* Returns: `{Array|Object}` the filtered object
+
+
+### fluid.censorKeys(toCensor, keys)
+
+A convenience wrapper for `fluid.filterKeys` with the argument `exclude` set to `true`. Returns a shallow clone of the supplied object with listed keys removed.
+
+* `toFilter {Array|Object}` The object to be filtered - this will be NOT modified by the operation
+* `keys {Array of String}` The list of keys to operate with
+* Returns: `{Array|Object}` A shallow clone of `toFilter` with the supplied keys removed
+
+
+### fluid.keys(object)
+
+Return the keys in the supplied object as an array. Note that this will return keys found in the prototype chain as well as "own properties", unlike the builtin [`Object.keys()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)
+
+* `object {Object}` The object to have its keys listed
+* Returns: `{Array}` An array holding the keys of this object.
+
+
+### fluid.values(object)
+
+Return the keys in the supplied object as an array. This will return values found in the prototype chain as well as those attached to "own properties".
+
+* `object {Object}` The object to have its values listed
+* Returns: `{Array}` An array holding the values of this object.
+
+
+### fluid.arrayToHash(array)
+    
+Converts an array into an object whose keys are the elements of the array, each with the value `true`
+* `array {Array of String}` The array to be converted to a hash
+* Returns: `{Object}` An object with value `true` for each key taken from a member of `array`
+
+
+### fluid.hashToArray(hash, keyName[, func])
```

Comment:
The description of hashToArray(), including the parameters, needs a bit of clarification. Very not clear... I can't really suggest improvements, since I'm totally confused as to what it does.